mungegithub/mungers/issue-cacher.go

```diff
@@ -70,6 +70,7 @@ func (p *IssueCacher) RequiredFeatures() []string { return []string{} }
 
 // Initialize will initialize the munger
 func (p *IssueCacher) Initialize(config *github.Config, features *features.Features) error {
+	// TODO: this need to be changed
```

Comment:
Comments like this are more useful if they state what about it needs to be changed :)